cpp/src/io/json/json_tree.cu

```diff
@@ -125,6 +128,70 @@ struct node_ranges {
   }
 };
 
+/**
+ * @brief Returns stable sorted key and its sorted order
+ *
+ * Uses cub stable radix sort
+ *
+ * @tparam IndexType sorted order type
+ * @tparam KeyType key type
+ * @param key key to sort
+ * @param stream CUDA stream used for device memory operations and kernel launches.
+ * @return A pair of sorted key and its sorted order
+ */
+template <typename IndexType = size_t, typename KeyType>
+std::pair<rmm::device_uvector<KeyType>, rmm::device_uvector<IndexType>> stable_sorted_key_order(
+  cudf::device_span<KeyType const> key, rmm::cuda_stream_view stream)
+{
+  CUDF_FUNC_RANGE();
+
+  // Determine temporary device storage requirements
+  rmm::device_uvector<TreeDepthT> key1(key.size(), stream);
+  rmm::device_uvector<TreeDepthT> key2(key.size(), stream);
```

Comment:
you don't seem to be using the template parameter? Maybe this should be
```suggestion
  rmm::device_uvector<KeyType> key1(key.size(), stream);
  rmm::device_uvector<KeyType> key2(key.size(), stream);
```

Reply:
missed it, while moving code. Thanks. 👍  updated.